.github/check-translations.php

```diff
@@ -0,0 +1,86 @@
+<?php
```

Comment:
```suggestion
<?php

declare(strict_types=1);
```

Reply:
we don't use strict types as you might know.

Reply:
I know for whole project it's challenging, but for standalone script, I would recommend it ;)

Reply:
strict mode is not safer than the coercive mode, especially in modern PHP version where unsafe coercion rules have been removed or deprecated.
Strict mode tend to force you to use explicit casting instead. And explicit casting is **very** unsafe as it will happily perform operations that lose information totally silently. `(int) "I'm not a number"` will give you `0` without any warning, while an implicit coercion by a non-strict-mode parameter will reject it)

Reply:
ha, i was actually preventing any casting in my codebase, but I guess I do not handle that much user input.